File: TTT_UWP/TTT_UWP.Model/Region.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TTT_UWP.Model
{
    public class Region
    {
        public int RegionID { get; set; }
        public string RegionName { get; set; }
        public int WarehouseID { get; set; }
        public int SensorID { get; set; }
        public Observation[] Observations { get; set; }
        public Rack[] Racks { get; set; }
    }
}

File: api/src/main/java/be/ttt/api/entities/Region.java
package be.ttt.api.entities;

import java.util.ArrayList;
import java.util.List;

import javax.persistence.*;

import com.fasterxml.jackson.annotation.JsonIgnore;

@Entity
@Table(name = "regions")
public class Region {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;
	private String name;
	private int warehouseId;

	@OneToMany(mappedBy="regionId")
	@JsonIgnore
	private List<Rack> racks = new ArrayList<Rack>();
	
	@OneToMany(mappedBy="regionId")
	@JsonIgnore
	private List<Observation> observations = new ArrayList<Observation>();

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public int getWarehouseId() {
		return warehouseId;
	}

	public void setWarehouseId(int warehouseId) {
		this.warehouseId = warehouseId;
	}

	public List<Rack> getRacks() {
		return racks;
	}

	public void setRacks(List<Rack> racks) {
		this.racks = racks;
	}

	public List<Observation> getObservations() {
		return observations;
	}

	public void setObservations(List<Observation> observations) {
		this.observations = observations;
	}
	
	
}